### DISCUSSION TITLE
How to protect your token?

### DISCUSSION THREAD

**Discussion by RINCODER** [Eptagone/Telegram.BotAPI#51] [Q&A]
Greetings, I am using your library (package) to send notifications in a telegram, but I ran into such a problem that my token is also sent in network traffic when sending a message, and literally a day ago my token was thrown off from the bot.

Is there a way to protect the token? The program is located with different people and I would not like it to get to them in

(without server)

**Comment by Eptagone** [comment]
If I understood correctly, you are sharing the program with the token to different people, right?

I don't know why you do it but that's wrong. Your bot must be on a server where only you can access information like the token.
